terraform/electionleaflets-dns.tf

```diff
@@ -3,39 +3,11 @@ resource "cloudflare_record" "el_root" {
   domain = "electionleaflets.org.au"
   name   = "electionleaflets.org.au"
   type   = "A"
-  value  = "${aws_eip.kedumba.public_ip}"
-}
-
-resource "cloudflare_record" "el_ec2" {
```

Comment:
I wouldn't remove the ec2 records until after the migration is done. Then you don't have to change so much in one go.